.github/CONTRIBUTION/CONTRIBUTION.md

```diff
@@ -14,29 +14,27 @@
 
 * Clone or download the repo. into any fresh temporary folder.
 
-    ``` git clone https://github.com/Ignitus/Ignitus-Client-Side-Development.git ```
+    ``` git clone https://github.com/Ignitus/Ignitus-client.git ```
```

Comment:
@divyanshu-rawat is there any reason to do so? Because ```Ignitus-Client-Side-Development``` redirects to ```Ignitus-client```.

Reply:
@gittysachin good point, but still its good practise to rename it , as it may confuse 😕 some new comers they will be like 🤷‍♂️ 🤷‍♀️.